src/routes/schema/userSchema.js

```diff
@@ -2,9 +2,9 @@ import Joi from 'joi';
 
 export const USER_EDIT_SCHEMA = Joi.object().keys({
   name: Joi.string().max(255).required(true),
-  id: Joi.number(),
+  id: Joi.string().max(255).required(true),
```

Comment:
1. You would validate for max, min, allow characters.
2. name and id are the only available pos params

Reply:
noted.